test/format/backup.c

```diff
@@ -457,6 +457,58 @@ save_backup_info(ACTIVE_FILES *active, uint64_t id)
     free(to_path);
 }
 
+/*
+ * copy_format_files --
+ *     Copies over format-specific files to the BACKUP.copy directory.
+ */
+static void
+copy_format_files(WT_SESSION *session)
+{
+    size_t dest_len, file_len;
+    u_int i;
+    char *dest, *filename;
+    bool exist;
+
+    /* Copy over the CONFIG file. */
+    dest_len = strlen("BACKUP.copy") + strlen("CONFIG") + 10;
+    dest = dmalloc(dest_len);
+    testutil_check(__wt_fs_exist((WT_SESSION_IMPL *)session, "CONFIG", &exist));
+    if (exist) {
+        testutil_check(__wt_snprintf(dest, dest_len, "BACKUP.copy/%s", "CONFIG"));
+        testutil_check(__wt_copy_and_sync(session, "CONFIG", dest));
+    }
+    free(dest);
```

Comment:
While this is correct, I think what you want to do here is generate just the string name like "CONFIG" and call `testutil_copy_file` instead. That already takes care of everything. If the existence is an issue, you can add a boolean arg to that function to ignore ENOENT errors. `test/format/backup.c` is the only caller of `testutil_copy_file`.

So something like `testutil_copy_file(session, "CONFIG", true);` and `testutil_copy_file(session, "CONFIG.keylen", false);` and then something like this in `test/utility/misc.c`: 
```testutil_copy_file(..., bool must_exist)
...
ret = __wt_copy_and_sync(...)
if (must_exist)
    testutil_check(ret);
else
    testutil_assert(ret == 0 || ret == ENOENT);
```
(Note this may not be 100% code-correct but just to get the idea.)